html/cross-origin-opener-policy/coep-navigate-popup.https.html

```diff
@@ -49,11 +49,12 @@
     const channel = title.replace(/ /g,"-");
     const navigateHost = site === "same-origin" ? SAME_ORIGIN : SAME_SITE;
     const navigateURL = `${navigateHost.origin}/html/cross-origin-opener-policy/resources/coop-coep.py?coop=${variant.coop}&coep=${variant.coep}&channel=${channel}`;
-    const opener = site === "same-origin" ? variant.opener : false;
+    const opener = site === "same-origin" ? variant.opener : true;
 
     async_test(t => {
       // For each test we open a COOP: same-origin/COEP: require-corp document in a popup and then
-      // navigate that to either a same-origin or same-site document whose COOP and COEP are set as
+      // navigate that to either a same-origin or "same-site" (a historical value which today should
```

Comment:
Nit: I would slightly prefer statements of fact throughout, i.e., "is equivalent", but it's no big deal so not blocking.

Reply:
Actually I think this comment is now confusing. The `site` value is a test internal thing to decide what to load in the popup, not a COOP value. Right? Maybe the values should be different to avoid confusion.

Reply:
Oh right, yeah. I think what I'd also prefer is to test the historical values separately in a single historical test if possible to make the tests less confusing for new implementers.

Reply:
Fixed this comment in https://github.com/web-platform-tests/wpt/pull/20874/commits/1aeef8bba02f36fdca37fb5f483ccd11b6dfd462 and changed other "should be equivalent"s to "is equivalent".

Reply:
I can move all historical parts into the historical/ directory. Maybe we don't need to test all aspects for historical values, and instead have a simpler test for them.

Reply:
Yeah, I don't think we need all aspects, but this also doesn't need to be perfect. So don't make more work for yourself than needed when it comes to the historical tests.

Reply:
Yeah, that makes sense. I've done part of it now, and all historical aspects are marked with comments so it can be understood reasonably now and refactored later to separate them more. Thx!